Warnings in btl/ofi

Current head of master, on Mac OSX:
```shell
btl_ofi_frag.c:152:5: warning: static function 'mca_btl_ofi_frag_complete' is used in an inline function with external
      linkage [-Wstatic-in-inline]
    mca_btl_ofi_frag_complete(frag, OPAL_SUCCESS);
    ^
./btl_ofi_frag.h:44:1: note: use 'static' to give inline function 'mca_btl_ofi_recv_frag' internal linkage
int mca_btl_ofi_recv_frag (mca_btl_ofi_module_t *ofi_btl,
^
static 
./btl_ofi_frag.h:81:20: note: 'mca_btl_ofi_frag_complete' declared here
static inline void mca_btl_ofi_frag_complete (mca_btl_ofi_base_frag_t *frag, int rc) {
```

--- a/opal/mca/btl/ofi/btl_ofi_frag.c
+++ b/opal/mca/btl/ofi/btl_ofi_frag.c
@@ -132,7 +132,7 @@ int mca_btl_ofi_send (mca_btl_base_module_t *btl,
     return OPAL_SUCCESS;
 }
 
-inline int mca_btl_ofi_recv_frag (mca_btl_ofi_module_t *ofi_btl,
+int mca_btl_ofi_recv_frag (mca_btl_ofi_module_t *ofi_btl,
                            mca_btl_base_endpoint_t *endpoint,
                            mca_btl_ofi_context_t *context,
                            mca_btl_ofi_base_frag_t *frag)

